diamm/serializers/website/source.py

```diff
@@ -68,11 +68,14 @@ def get_related_entity(self, obj):
 
 
 class SourceProvenanceSerializer(ContextDictSerializer):
+    entity = serpy.MethodField()
+    protectorate = serpy.MethodField()
+    region = serpy.MethodField()
     city = serpy.MethodField()
     country = serpy.MethodField()
-    region = serpy.MethodField()
-    protectorate = serpy.MethodField()
-    entity = serpy.MethodField()
+
+    class Meta:
+        ordering = ('entity', 'protectorate', 'region', 'city', 'country')
```

Comment:
Does this actually work? I can't find it in the documentation... 

http://serpy.readthedocs.io/en/latest/index.html

Are you trying to control the order of the output fields?